oxidation/crates/parsec_api_protocol/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+mod block;
+mod events;
+mod invite;
+mod message;
+mod organization;
+mod ping;
+mod realm;
+mod user;
+mod vlob;
+
+pub use block::*;
+pub use events::*;
+pub use invite::*;
+pub use message::*;
+pub use organization::*;
+pub use ping::*;
+pub use realm::*;
+pub use user::*;
+pub use vlob::*;
+
+use serde::{Deserialize, Serialize};
+
+#[macro_export]
```

Comment:
no need for the macro_export given the macro won't be used outside of the crate

you can add a `pub(crate) use impl_api_protocol_dump_load;` after the macro definition instead (cf. `new_uuid_type` macro)